src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -375,6 +375,23 @@ public void ClearRenderEffect(bool disposeEffect = false)
             }
         }
 
+        /// <summary>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ImageUrl GetImageUrl()
```

Comment:
```suggestion
        public ImageUrl GetOffscreenImageUrl()
```

Please make the name that this is only for `Offscreen`. Current name looks so general.

Also please remain summary that this will return valid ImageUrl only if we use Once option.